src/modules/keycloak-administration/domain/keycloak-client/user-representation.dto.ts

```diff
@@ -15,4 +15,9 @@ export class UserRepresentationDto {
 
     @IsNumber()
     public createdTimestamp!: number;
+
+    @IsBoolean()
+    public enabled!: boolean;
+
+    public attributes!: Record<string, string>;
 }
```

Comment:
If I recall correctly, the attributes returned by keycloak are actually `Record<string, string[]>`.
Could you double-check this?